frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/MergeCurrentBranchFastForwardOnlyBackgroundable.java

```diff
@@ -31,8 +33,8 @@ public MergeCurrentBranchFastForwardOnlyBackgroundable(
   }
 
   @Override
-  protected @I18nFormat({}) String getOperationName() {
-    return getString("action.GitMachete.MergeCurrentBranchFastForwardOnlyBackgroundable.operation-name");
+  protected @I18nFormat({}) @Untainted String getOperationName() {
+    return getNonHtmlString("action.GitMachete.MergeCurrentBranchFastForwardOnlyBackgroundable.operation-name");
```

Comment:
To be considered, not insisting:

```suggestion
    return getString("action.GitMachete.MergeCurrentBranchFastForwardOnlyBackgroundable.operation-name").nonHtml();
```

Plus also, require that each property that contains HTML, has a `.HTML` "extension", e.g.:

```properties
action.GitMachete.OpenMacheteTabAction.notification.title.could-not-open-tab.HTML=<html><b>Could not open Git Machete tab</b></html>
```

This way one could easily spot sth like:

```java
getString("action.GitMachete.OpenMacheteTabAction.notification.title.could-not-open-tab.HTML").nonHtml() // WHOOPS
```

Reply:
(`nonHtml` could be ofc an extension method declared in `GitMacheteBundle`, similarly to `format`)

Reply:
(and `getNonHtml` could be removed completely then)

Reply:
that sounds good to me but in this case together with `format` it would look like `getString("...").nonHtml().format("...")`. Would it be clear enough ?

Reply:
> where html may be written but we do not use this possibility

In such places, I would NOT append `.HTML` &mdash; only where HTML is actually used.
I'd separate **properties** itself from the **methods/parameters** to which they're passed (esp. that any given non-HTML property can be passed to both `@Untainted` and `@Tainted` params).
In other words, properties themselves should **not** care that to what params they're passed &mdash; only whether they contain HTML or not

Reply:
> (`nonHtml` could be ofc an extension method declared in `GitMacheteBundle`, similarly to `format`)

I have problem with this implementation because `format ` expect proper `i18nFormat` (which is at this moment generating from `getString` because we have `@ I18nMakeFormat` there). I am trying to cheat checkerframework but with no effect so far

Reply:
Ouch, I forgot about this limitation 🤦🏻

Indeed, I don't think this is possible to circumvent (as in, I don't think you'll be able to mark `nonHtml`'s result as a format string)... so sticking to `getNonHtmlString` might be a better idea

Reply:
But `.HTML` suffix for property keys still makes sense IMO